packages/amplify-storage-simulator/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "amplify-storage-simulator",
+  "version": "0.0.1",
+  "description": "An S3 simulator to test S3 APIs",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "start": "node lib/test/test.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@hapi/topo": "^3.1.2",
+    "@types/body-parser": "^1.17.0",
+    "@types/cors": "^2.8.5",
+    "@types/glob": "^7.1.1",
+    "@types/serve-static": "^1.13.2",
+    "body-parser": "^1.19.0",
+    "chokidar": "^3.0.2",
```

Comment:
Do you need this?

Reply:
yes , I need glob, serve-static , bodyparser but not chokidar